src/app/_utilities/formatDateTime.ts

```diff
@@ -0,0 +1,14 @@
+export const formatDateTime = (timestamp: string): string => {
+  const now = new Date();
+  let date = now;
```

Comment:
do we need the const now = new Date()?

could we just do `let date = new Date();`